Content.Server/Procedural/DungeonJob/DungeonJob.OreDunGen.cs

```diff
@@ -20,44 +21,52 @@ private async Task PostGen(
     {
         // Doesn't use dungeon data because layers and we don't need top-down support at the moment.
 
-        var emptyTiles = false;
         var replaceEntities = new Dictionary<Vector2i, EntityUid>();
         var availableTiles = new List<Vector2i>();
+        var tiles = _maps.GetAllTilesEnumerator(_gridUid, _grid);
 
-        foreach (var node in dungeon.AllTiles)
+        while (tiles.MoveNext(out var tileRef))
```

Comment:
This probably needs some warning on it because if you're generating a random dungeon then you might inadvertantly run this across a whole map.

Reply:
Done